README.md

```diff
@@ -68,9 +74,51 @@ After writing this method, our new result should be:
 ```
 
 **think**: Where should we call `strip_punctuation`? Does it matter? Why?
+This method should be called right before the split method because words with punctuation will not be evaluated correctly if it is not applied.
```

Comment:
Yes, I should have also asked "Should strip_punctuation be called inside or outside the analyze_mood method?" You put it inside, which is what I wanted. But I wanted to hear your thinking on why that's the right choice.